wlib/RepositoryView.h

```diff
@@ -4,6 +4,7 @@
 #include "Repository.h"
 #include "RepositoryTreeNode.h"
 #include <Thread.h> //clib
+#include <EclCC.h>
```

Comment:
@GordonSmith you get this error: 'IsLocalRepositoryEnabled': identifier not found if you do not include that header file.